### ISSUE TITLE
ibm_is_instance 1.33.0 no longer creates user_data

### ISSUE THREAD

**Issue by jgruberf5** [IBM-Cloud/terraform-provider-ibm#3163]
### Terraform CLI and Terraform IBM Provider Version

Schematics - terraform 0.14

2021/09/30 14:16:43 Starting command: terraform14 init -input=false -lock=false -no-color

### Affected Resource(s)

* ibm_is_instance

### Terraform Configuration Files

Create an instance... below image ID is Ubuntu 20.04, use whatever profile, vpc, subnet, security groups you want..

```hcl

terraform {
  required_providers {
    ibm = {
      source  = "IBM-Cloud/ibm"
    }
  }
}

resource "ibm_is_instance" "test_instance" {
  name           = "test-instance"
  resource_group = "default"
  profile        = data.ibm_is_instance_profile.instance_profile.id
  image          = "r006-988caa8b-7786-49c9-aea6-9553af2b1969"
  primary_network_interface {
    name              = "outside"
    subnet            = data.ibm_is_subnet.outside_subnet.id
    security_groups   = [data.ibm_is_vpc.volterra_vpc.default_security_group]
    allow_ip_spoofing = true
  }
  vpc       = data.ibm_is_subnet.outside_subnet.vpc
  zone      = data.ibm_is_subnet.outside_subnet.zone
  keys      = [data.ibm_is_ssh_key.ssh_key.id]
  user_data = "#cloud-config\nwrite_files:\n- path: /etc/test.flag\n  permissions: '0644'\n  content: |\n
    cloudinit wrote this"
  timeouts {
    create = "60m"
    delete = "120m"
  }
}
```

### Expected Behavior

What is put in the user_data attribute of ibm_is_instance should be sent via cloud-init.

### Actual Behavior

Instance starts, but the user-data cloudinit file is empty.

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

We started to get reports that our Schematic github workspaces stopped working properly. After examination of the VSIs it was clear that cloudinit user-data was not being populated in the NoCloud data source attached volume. We reproduced with generic Ubuntu 20.04 instances with hard coded string user_data attributes on ibm_is_instance. See above HCL.

Changing the provider to:

```hcl
terraform {
  required_providers {
    ibm = {
      source  = "IBM-Cloud/ibm"
      version = "1.30.0"
    }
  }
}
```
in out workspaces fixed the issues.

These issues started when 1.33.0 was added to the terraform registry and was used in Schematics (terraform init). 

### References

* #0000

**Comment by ujjwal-ibm** [comment]
@jgruberf5 We have identified the issue and fix, please use 1.32.1 or lower to unblock yourself

**Comment by kavya498** [comment]
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/3160

**Comment by kavya498** [comment]
Available in latest 1.33.1
Closing this issue thanks..
